core-ui/src/shared/ResourceForm/components/ResourceForm.scss

```diff
@@ -6,6 +6,23 @@
   form {
     overflow: auto;
     flex-grow: 1;
+    order: 2;
+  }
+
+  &:before {
+    content: '';
+    width: 100%;
+    height: 16px;
+    background: linear-gradient(
```

Comment:
I'm not an ux designer, but shouldn't this gradient be applied also to the bottom of a scrollable list?

Reply:
Maria only wanted it at the top :shrug:

Reply:
Then only top it is.